### PR TITLE
Add basic e2e test

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/giantswarm/issues/4076

Same change as https://github.com/giantswarm/cert-exporter/pull/18. See the comments there for the Circle magic to support deploying with draughtsman.

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -25,26 +25,67 @@
...
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/e2e-harness"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/e2esetup"
+
+[[constraint]]
+  branch = "release-tests-flag"
```
Comment (rossf7) [inline Gopkg.toml]: TODO

**Comment by kopiczko** [comment]
@giantswarm/team-batman @giantswarm/sig-operator: should that be just merged?
I think this is the last component missing to close https://github.com/giantswarm/giantswarm/issues/4076.

**Comment by xh3b4sd** [comment]
I would say lets go when nothing is missing.

**Comment by kopiczko** [comment]
This is all green. Merging.

**Pull request merged**